core-services/ci-chat-bot/workflows-config.yaml

```diff
@@ -257,6 +257,8 @@ workflows:
     platform: openstack
   openshift-e2e-vsphere:
     platform: vsphere
+  openshift-e2e-vsphere-ovn-dualstack:
```

Comment:
Is it the right one you want, did you mean to add `cucushift-installer-rehearse-vsphere-ipi-ovn-dualstack` ?

Reply:
` cucushift-installer-rehearse-vsphere-ipi-ovn-dualstack` is already added.  We want both eventually.  Everyone should be testing dualstack, it's the future.

Reply:
Ack